### DISCUSSION TITLE
Automatically determine notable contributions

### DISCUSSION THREAD

**Discussion by jayvdb** [lowlighter/metrics#588] [Ideas]
https://github.com/ghuser-io/ghuser.io (defunct) had a very clever approach to score & order users contributions based on notability which I think would be very helpful to re-use here.  Ping @AurelienLourot & @brillout

Calculation per repo was explained as = user commit count * repo star boost * repo contrib boost.

See "Ranking" on https://github.com/ghuser-io/ghuser.io/issues/156 for more details.

See https://github.com/ghuser-io/ghuser.io/blob/master/docs/screenshot.png for an example

This would need a similar "indepth" data collection in order to compute the stats, as https://github.com/lowlighter/metrics/tree/master/source/plugins/languages#using-indepth-statistics

c.f. https://github.com/lowlighter/metrics/discussions/550 for related feature request for manually specifying the notable contributions.

**Comment by lowlighter** [comment]
This looks pretty interesting 👍 
Will take a look later and check later how this can be implemented within Metrics.

Thanks for your suggestion 🙂 

(tracked in #615)

**Comment by jayvdb** [comment]
https://codetrace.com has another approach which (IMO) isnt as good as ghuser.io.

But might be useful for ideas and comparison https://codetrace.com/users/jayvdb

**Comment by lowlighter** [comment]
I think it kind of match the additional "badges" that were introduced in the notable plugin. 
You can now display "relative" commits/issues/pr/stars badges 

I'll mark this as shipped
